modules/ROOT/pages/bonita-bpm-studio-installation.adoc

```diff
@@ -7,32 +7,11 @@ How to install a Bonita Studio on Windows, Linux, or Mac operating systems. An O
 
 Check the xref:hardware-and-software-requirements.adoc[hardware and software requirements].
 
-Before you download Bonita Studio, make sure that you know whether you are using a using a 32 or 64-bit system, and that you have the appropriate Java version installed:
+=== Macos specific prerequisites
 
-* For Linux, to find out whether you are using a 32 or 64 bit Linux, run the following command: `getconf LONG_BIT`, which returns either 32 or 64.
-* For Windows, see the http://windows.microsoft.com/en-us/windows/32-bit-and-64-bit-windows[32 and 64 bit Windows FAQ].
-* For Mac, 32-bit Java is no longer supported, so there is no 32-bit version of Bonita for Mac systems.
-
-[IMPORTANT]
-====
-
-Both Windows and Mac have default security settings that will prevent execution of Bonita. See below for further details about what you can do to bypass those security protections
-====
-
-*Note for users of macOS 10.12.x and above* : There is an known issue on macOS Sierra and Java about the slowness of   java.net.InetAddress getLocalHost() which results in a slowness of the Bonita Studio (find more info on https://thoeni.io/post/macos-sierra-java/[thoeni] or https://plumbr.eu/blog/java/macos-sierra-problems-with-java-net-inetaddress-getlocalhost[plumbr]). To resolve this issue you shoud add your computer name to your /etc/hosts file : In a terminal, edit your `/etc/hosts` with sudo privilege, add your computer name to the local IP addresses `127.0.0.1 localhost <mycomputername.local>` and `::1 localhost <mycomputername.local>` (To find your macOS computer name, look at https://support.apple.com/kb/PH25076[Apple support dedicated page]).
-
-*Notes for users of macOS Catalina 10.15 and above*:
-[IMPORTANT]
-====
-
-Only version 7.7.5 and upwards are compatible
-====
-
-*Notes for users of macOS Big Sur 11.0 and above*:
 [IMPORTANT]
 ====
-
-Only version 7.11.4 and upwards are compatible
+For users of macOS Big Sur 11.0 and above: Only version 7.11.4 and upwards are compatible
```

Comment:
❓ not specific to 2021.1, how can we put this information in previous versions?

Reply:
It is already present in previous version, I just clean the page and rewrote some stuff, with latest informations.